plugins/analytics-module-segment/config.d.ts

```diff
@@ -0,0 +1,72 @@
+export interface Config {
+  app: {
+    analytics: {
+      segment: {
+        /**
+         * Controls whether the Segment Analytics module is enabled.
+         * Defaults to true.
+         *
+         * @visibility frontend
+         */
+        enabled?: boolean;
+
+        /**
+         * Segment Analytics Write Key. Reference https://segment.com/docs/connections/find-writekey/
+         * to find your write key
+         *
+         * @visibility frontend
+         */
+        writeKey: string;
+
+        /**
+         * Whether to log analytics debug statements to the console.
```

Comment:
and also write analytics events to the web console (according to the README)

Reply:
Resolved in https://github.com/segmentio/segment-backstage-plugins/pull/12/commits/d39a9ca380fa3af95719bd9055a22cdf92d317ec